variables.tf

```diff
@@ -229,3 +198,60 @@ variable "manager_pod_annotations" {
   default     = {}
 }
 
+variable "additional_secrets" {
+  description = "additional secrets to mount into the manager pods"
+  type        = list(map(string))
+  default     = []
+}
+
+variable "replicas" {
+  description = "the number of manager pods to create"
+  type        = number
+  default     = 1
+}
+
+variable "runner_name" {
+  description = "name of the runner"
+  type        = string
+}
+
+variable "unregister_runners" {
+  description = "whether runners should be unregistered when pool is deprovisioned"
+  type        = bool
+  default     = true
+}
+
+variable "runner_token" {
+  description = "token of already registered runer. to use this var.runner_registration_token must be set to null"
+  type        = string
+  default     = null
+}
+
+
+variable "cache" {
+  description = "value"
```

Comment:
Add a very vivid description that emcompasses what the cache variable is and does.